src/components/FeatureExportButton/README.md

```diff
@@ -20,22 +21,40 @@ import FeatureExportButton from 'react-spatial/components/FeatureExportButton';
 const vectorLayer = new Layer({
   olLayer: new VectorLayer({
     style: new Style({
-      image: new Icon({
-        anchor: [0.5, 46],
-        anchorXUnits: 'fraction',
-        anchorYUnits: 'pixels',
-        src: 'https://openlayers.org/en/latest/examples/data/icon.png',
-        size: [32, 48]
-      })
+      image: new CircleStyle({
+        stroke: new Stroke({
+          color: '#3399CC',
+          width: 4,
+        }),
+        fill: new Fill({
+          color: '#3399CC',
+          width: 4,
+        }),
+      }),
+      // image: new Icon({
+      //   anchor: [0.5, 46],
```

Comment:
Maybe clean these. Don't see the point of keeping them

Reply:
I put back the original example